CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Rewrite tests which depends on stripes-data-transfer-components.Refs UIDEXP-380.
 * Improve field suppression validation. Refs UIDEXP-378.
 * Error message  in error " Logs", if the user does not have the affiliation with the tenant. Refs UIDEXP-381.
+* Add consistency to Add mapping profile form. Refs UIDEXP-382.
```

Comment:
This could probably re-worded to emphasize what consistency we are achieving ex- Consistent info icons on ...